test/norm_swap.lean

```diff
@@ -0,0 +1,95 @@
+import data.equiv.basic
+import tactic.norm_swap
+
+
+open equiv tactic
+
+local attribute [-simp] swap_apply_left
+local attribute [-simp] swap_apply_right
```

Comment:
Are these just needed because of how the test is set up? (If so could you add a short comment?) Or are they needed to make the plugin work in practice?

Reply:
Turning off the simp tags makes sure we're testing the tactic, and not the simp-using capabilities of `norm_num`. Comment added.